server/db/seeders/user.js

```diff
@@ -0,0 +1,42 @@
+require('dotenv').config()
+const faker = require('faker');
+const { User, db } = require('../db')
+const encryption = require('../../helpers/encryption');
+
+/**
+ * Method to generate fake users
+ * @param {number} amount  Request object
+ * @return {array} array of fake user object generated
+ */
+async function generateFakeUsers(amount = 10) {
+  const fakeUsers = [];
+  for (let i = 0; i <= amount; i++) {
+    fakeUsers.push({
+      firstName: faker.name.firstName(),
+      lastName: faker.name.lastName(),
+      email: faker.internet.email(),
+      password: await encryption.hashPassword('sweetlove'),
+      phone: faker.phone.phoneNumberFormat()
+    });
+  }
+  console.log(fakeUsers);
```

Comment:
Any particular reason for console.log here?

Reply:
No, I would remove it now